app/views/application/_browsing_index.html.erb

```diff
@@ -1,6 +1,7 @@
 <div class="container">
-<% cache(action: "index", action_suffix: "obituaries_index") do %>
+
   <% @obituaries.each do |obit| %>
+  <% cache(obit) do %>
```

Comment:
Does this work on your local host?

Reply:
yes

Reply:
with rails dev:cache
Development mode is now being cached.

Reply:
Okay sounds good.